blueprint/client-devices.md

```diff
@@ -895,89 +895,371 @@ Microsoft Defender Smart Screen Configuration | Enabled and configured | Meets A
+    * Deny read access: Disabled
+    * Deny write access: Enabled
+  * Custom Classes:
+    * Deny read access: Disabled
+    * Deny write access: Enabled
+  * Floppy Drives:
+    * Deny execute access: Enabled
+    * Deny read access: Disabled
+    * Deny write access: Enabled
+  * Removable Disks:
+    * Deny execute access: Enabled
+    * Deny read access: Disabled
+    * Deny write access: Enabled
+  * Tape Drives:
+    * Deny execute access: Enabled
+    * Deny read access: Disabled
+    * Deny write access: Enabled
+  * WPD Devices:
+    * Deny read access: Disabled
+    * Deny write access: Enabled
+* File and print sharing
+  * Justification: To align with the ACSC Windows 10 hardening guidance.
+  * Prevent the computer from joining a homegroup: Enabled
+  * Prevent users from sharing files within their profile: Enabled
+* Hard drive encryption
+  * Justification: To align with the ACSC Windows 10 hardening guidance, with the exception of pre-boot PIN for useability.
+  * BitLocker Drive Encryption: Enabled
+  * Disable new DMA devices when this computer is locked: Enabled
+  * Prevent memory overwrite on restart: Disabled
+  * Fixed Data Drives: Enabled
+  * Allow network unlocked at startup: Enabled
+  * Allow Secure Boot for integrity validation: Enabled
+  * Enforce drive encryption type on operating system drives: Enabled (Full encryption)
+  * Reset platform validation data after BitLocker recovery: Enabled
+  * Removable Data Drives: Enabled
+  * Control use of BitLocker on removable drives: Enabled (Allow users to apply BitLocker protection on removable data drives)
+  * Deny write access to removable drives not protected by BitLocker: Enabled
+  * Enforce drive encryption type on removable data drives: Enabled (Full encryption)
+  * Pre-boot PIN: Disabled
+* Installing applications and drivers
+  * Justification: To align with the ACSC Windows 10 hardening guidance.
+  * Configure Windows Defender SmartScreen: Enabled (Warn and prevent bypass)
+  * Allow user control over installs: Disabled
+  * Always install with elevated privileges: Disabled
+  * Devices: Prevent users from installing printer drivers: Enabled
+  * Always install with elevated privileges: Disabled
+* Legacy and run once lists
+  * Justification: To align with the ACSC Windows 10 hardening guidance.
+  * Do not process the legacy run list: Enabled
+  * Do not process the run once list: Enabled
+  * Run these programs at user logon: Disabled
+* Microsoft accounts
+  * Justification: To align with the ACSC Windows 10 hardening guidance.
+  * Block all consumer Microsoft account user authentication: Enabled
+  * Prevent the usage of OneDrive for file storage: Enabled
+  * Accounts: Block Microsoft accounts: Users can’t add or log on with Microsoft accounts
+* MSS settings
+  * Justification: To align with the ACSC Windows 10 hardening guidance.
+  * IP source routing protection level: Enabled
+  * IP source routing protection level: Enabled
```

Comment:
These appear twice and were there in the original formatting. Is there a missing setting or incorrect double paste?